composer.json

```diff
@@ -5,6 +5,9 @@
     "license": "MIT",
     "homepage": "https://laravel.com",
     "support": {
+        "forum": "https://laravel.com/forum",
+        "chat": "https://laravel.com/chat",
```

Comment:
The forum and chat links load error pages.

![image](https://user-images.githubusercontent.com/5625680/155339463-0636381c-ac55-4e67-ad45-9acbdb11f098.png)

Reply:
Please read the description and check the linked PR